model/docs/getting_started.qmd

```diff
@@ -10,7 +10,7 @@ engine: jupyter
 
 This tutorial assumes some pre-existing knowledge of infectious disease dynamics and Python programming. Before you dive in, we recommend:
 
- - Installing Python3 (use tools like [pyenv](https://realpython.com/intro-to-pyenv/) or [compile and install](https://ubuntuhandbook.org/index.php/2023/05/install-python-3-12-ubuntu/) from the [source page](https://www.python.org/downloads/))
+ - Verifying you have Python3.12 installed. If using wsl on VAP or GFE, the default python interpretor is likely python3.10 but pyrenew requires python3.12 (use tools like [pyenv](https://realpython.com/intro-to-pyenv/) or [compile and install](https://ubuntuhandbook.org/index.php/2023/05/install-python-3-12-ubuntu/) from the [release page](https://www.python.org/downloads/))
```

Comment:
Suggest avoiding language that's specific to internal users in our external docs

Reply:
I think the previous version (that doesn't mention 3.12 specifically) would be acceptable.

Reply:
Got it!